protobuf/src/test/java/com/linecorp/armeria/server/protobuf/ProtobufResponseAnnotatedServiceTest.java

```diff
@@ -218,8 +218,8 @@ void protobufJsonResponse(String contentType) throws InvalidProtocolBufferExcept
     void protobufStreamResponse(String path) {
         final AggregatedHttpResponse response = client.get(path).aggregate().join();
         assertThat(response.status()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR);
-        assertThat(cause).isInstanceOf(IllegalArgumentException.class)
-                         .hasMessageContaining("Cannot convert a");
+        assertThat(cause).isInstanceOf(IllegalStateException.class)
```

Comment:
Note to reviewers.
The exception is raised by `ProtobufResponseConverterFunction`. https://github.com/line/armeria/blob/master/protobuf/src/main/java/com/linecorp/armeria/server/protobuf/ProtobufResponseConverterFunction.java#L243-L244`
The exception is now wrapped by the `CompositeResponseConverterFunction`. 
https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/internal/server/annotation/CompositeResponseConverterFunction.java#L71-L75